python/samples/06.auth.oauth.messageExtensions/src/bot.py

```diff
@@ -0,0 +1,128 @@
+"""
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License.
+
+Description: initialize the app and listen for `message` activitys
+"""
+
+import sys
+import traceback
+import base64
+
+from typing import List, Any
+
+from msgraph import GraphServiceClient
+from botbuilder.core import MemoryStorage, TurnContext, CardFactory
+from botbuilder.schema import ThumbnailCard, CardImage
+from botbuilder.schema.teams import MessagingExtensionQuery, MessagingExtensionResult, MessagingExtensionParameter, MessagingExtensionAttachment, TaskModuleTaskInfo
+
+from teams import Application, ApplicationOptions, TeamsAdapter
+from teams.auth import AuthOptions, OAuthOptions
+from teams.state import ConversationState, TempState, TurnState, UserState
+
+from config import Config
+from graph import GraphTokenProvider
+
+config = Config()
+app = Application[TurnState[ConversationState, UserState, TempState]](
+    ApplicationOptions(
+        bot_app_id=config.APP_ID,
+        storage=MemoryStorage(),
+        adapter=TeamsAdapter(config),
+        auth=AuthOptions(
+            default="graph",
+            settings={
+                "graph": OAuthOptions(
+                    connection_name=config.OAUTH_CONNECTION_NAME,
+                    title="Sign In",
+                    text="please sign in",
+                    end_on_invalid_message=True,
+                    enable_sso=True,
+                ),
+            },
+        ),
+    )
+)
+
+
```

Comment:
tried poetry install for this sample and ran into `FileNotFound` for msgraph-sdk

Reply:
this must be an issue with your environment https://pypi.org/project/msgraph-sdk/